### PR TITLE
snort3: update to 3.1.50.0

### PR DESCRIPTION
**Pull request by graysky2**
Upstream bump
```
    Build system: x86_64
    Build-tested: bcm2711/RPi4B
    Run-tested: bcm2711/RPi4B
```

Signed-off-by: John Audia <therealgraysky@proton.me>

Maintainer: @flyn-org

### CODE REVIEW

File: net/snort3/files/local.lua [hunk 1]
```diff
@@ -50,3 +50,10 @@ file_policy = {
     }
   }
 }
+
```
Comment (BKPepe) [inline net/snort3/files/local.lua:53]: I would suggest to split these changes to separate commit.

Comment (graysky2) [reply net/snort3/files/local.lua:53]: done!

**Comment by graysky2** [comment]
As I mentioned in https://github.com/openwrt/packages/pull/20103, libdaq3 needs to be updated before this snort3 update will successfully build so the failure in the check is expected.

**Comment by flyn-org** [comment]
Please drop the "-1" from "snort3: update to 3.1.49.0-1". Other than this, the request looks good. Recommend merge.

**Comment by flyn-org** [comment]
It is frustrating to have a merge request change out from under me after I recommend merging it. Please revert this to the commit I reviewed. You can then submit another merge request to take the package to 3.1.50.0. I am grateful for your help, @graysky2, but I prefer we go about this in a more deliberate way.

**Comment by graysky2** [comment]
Ah, my bad.  I thought it made no sense to merge in the bump to 3.1.49.0 since it was superseded by 3.1.50.0 prior to getting merged.
